builder/builder.go

```diff
@@ -333,7 +333,7 @@ func (b *Builder) AddBundles(bundles []string, force bool, allbundles bool, git
 			}
 
 			fmt.Printf("Adding bundle %q\n", bundle)
-			if err = helpers.CopyFile(bundledir+bundle, clrbundledir+bundle); err != nil {
+			if err = helpers.CopyFile(bundledir+bundle, clrbundledir+bundle, true); err != nil {
```

Comment:
Instead of a boolean parameter, I would suggest one of the following to improve the code at the callsites

- Have two functions: CopyFile and CopyOverrideFile, or something along those lines. They should use a non-exported function that take e.g. the flags, so you don't duplicate implementation.
- Use flags instead of a boolean, so callsite can see either the word override or exclusive.
- Allow passing "extra" flags, so call sites would pass O_EXCL (kind of lazy version of the above).